namespaces/systemd.yml

```diff
@@ -151,6 +151,71 @@ namespace:
       description: >
         TODO
 
+    - name: AUDIT_FIELD_ACCT
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_ADDR
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_COMM
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_EXE
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_GRANTORS
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_HOSTNAME
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_OP
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_RES
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_SAUID
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_TERMINAL
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_UNIT
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_ID
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_TYPE
+      type: string
+      description: >
+        TODO
+
```

Comment:
I think these is when audit gets logged into journal.
We should probably move these to separate audit space at some point. I'm adding them for now, because they were hanging out there previously on the top level and we're working on reindexing the entire production ES cluster, so i want to keep track of these.